scripts/start.js

```diff
@@ -33,7 +34,8 @@ compiler.plugin('done', function (stats) {
 
     console.log(chalk.green('Compiled successfully!'));
     console.log('\nThe app is running at:');
-    console.log('\n    ' + chalk.cyan('http://localhost:' + port + '/'));
+    // TODO: display the correct port, if the default is busy.
+    console.log('\n    ' + chalk.cyan('http://localhost:' + DEFAULT_PORT + '/'));
```

Comment:
I have no opinion on promises and I've tried running app with your code, it works pretty well.

Should we go full-promise in `start.js` and should we add a prompt?